package.json

```diff
@@ -1,58 +1,59 @@
 {
-  "name": "react-oauth-popup",
-  "version": "0.1.1",
-  "description": "",
+  "name": "react-simple-oauth-popup",
```

Comment:
Are you also going to update the repository name?

Reply:
good catch, I already ran into it when releasing it haha.

Reply:
Fixed in master